client/src/positionUtil.js

```diff
@@ -0,0 +1,17 @@
+import LinkTo from "components/LinkTo"
+import _isEmpty from "lodash/isEmpty"
+import React from "react"
+import { ListGroup, ListGroupItem } from "react-bootstrap"
+
+export function getPositionsForRole(positions, role) {
+  const positionList = positions
+    ?.filter(pos => pos.role === role)
+    .map(pos => (
+      <ListGroupItem key={pos.uuid}>
+        <LinkTo modelType="Position" model={pos} />
```

Comment:
I imagine the users also want to see the person currently occupying this position here (so not only the position itself).

Reply:
Applied